contrib/storage-http/src/main/java/org/apache/drill/exec/store/http/udfs/HttpHelperFunctions.java

```diff
@@ -75,9 +75,6 @@ public void eval() {
       // as an approximation of null-if-null handling.
       if (args == null) {
         // Return empty map
-        org.apache.drill.exec.vector.complex.writer.BaseWriter.MapWriter mapWriter = writer.rootAsMap();
-        mapWriter.start();
```

Comment:
Do we still need the `writer` object in this class?

Reply:
We don't need it. but isince t is `@Output` it is required for the UDF signature and can't be removed. It can be checked one more time again once `complexWriters` are removed from all UDFs.
I've added todo to remove it. I can back to it, once migrating some other UDF to the new JSON parser